packages/salesforcedx-vscode-lwc/test/vscode-integration/metaSupport/MockRhExtension.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Extension, ExtensionKind, extensions } from 'vscode';
+import * as path from 'path';
+import { version } from 'process';
+
+export class MockRedhatExtension implements Extension<any> {
+  public extensionKind = ExtensionKind.Workspace;
+  constructor(version: string) {
+    this.id = 'redhat.vscode-xml';
+    this.extensionPath = 'extension/local/path';
```

Comment:
This might need to change and use `path.join` based on the comments I left on the test file.